pkg/resourceutils/updatereplicas.go

```diff
@@ -341,3 +345,94 @@ func getUpdatedReplicaCount(annotations map[string]string, activate bool, printF
 
 	return 0, false
 }
+
+func updateStashedCMObjects(namespace string, activate bool, printFunc func(string, string), config *rest.Config) {
+
+	newOwnerUIDMapping := make(map[string]types.UID)
+	// List the configmaps with label "stash-cr" enabled
+	configMaps, err := core.Instance().ListConfigMap(namespace, metav1.ListOptions{LabelSelector: "stash-cr"})
+	if err != nil {
+		util.CheckErr(err)
+		return
+	}
+	ruleset := resourcecollector.GetDefaultRuleSet()
+	// Create the CRs in the same namespace if those dont exist
+	for _, configMap := range configMaps.Items {
+		objBytes := []byte(configMap.Data["runtime-object"])
+		unstructuredObj := &unstructured.Unstructured{}
+		//err := json.Unmarshal(objBytes, &unstructuredObj)
+		err := unstructuredObj.UnmarshalJSON(objBytes)
+		if err != nil {
+			printFunc(fmt.Sprintf("Error converting string to Unstructured object: %s\n", err.Error()), "err")
+			continue
+		}
+		configClient, err := k8sdynamic.NewForConfig(config)
+		if err != nil {
+			util.CheckErr(err)
+			continue
+		}
+		//var client k8sdynamic.ResourceInterface
+		resource := &metav1.APIResource{
+			Name:       ruleset.Pluralize(strings.ToLower(unstructuredObj.GetKind())),
+			Namespaced: len(unstructuredObj.GetNamespace()) > 0,
+		}
+
+		var resourceClient k8sdynamic.ResourceInterface
+		if resource.Namespaced {
+			resourceClient = configClient.Resource(
+				unstructuredObj.GetObjectKind().GroupVersionKind().GroupVersion().WithResource(resource.Name)).Namespace(unstructuredObj.GetNamespace())
+		} else {
+			resourceClient = configClient.Resource(
+				unstructuredObj.GetObjectKind().GroupVersionKind().GroupVersion().WithResource(resource.Name))
+		}
+		//resourceInterface := configClient.Resource()
+		//client = resourceInterface.Namespace(namespace)
+
+		// Create the CR
+		_, err = resourceClient.Create(context.TODO(), unstructuredObj, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
If the error is already exists then should we log it on the CLI output?

Reply:
Will put a message hinting the resource exists.